sbc_compassion/tests/test_letter_import.py

```diff
@@ -0,0 +1,114 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp.tests import common
+
+import base64
+import csv
+import os
+import re
+
+IMPORT_DIR = os.path.join(os.path.dirname(__file__), "testdata/import")
+
+def get_file_content(path):
+    """ Reads a file and returns a base64 encoding of its contents. """
+    with open(path, "rb") as f:
+        file_content = f.read()
+        return base64.b64encode(file_content)
+
+def read_csv():
```

Comment:
These functions are quite specific to tests in this module, so I think they should stay where they are